soc/riscv/riscv-privilege/telink_b91/Kconfig.soc

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+prompt "CPU Architecture of SoC"
+depends on SOC_SERIES_RISCV_TELINK_B91
+
+config B91_CPU_RISCV32
+	bool "RISCV32 CPU Architecture"
+	
+endchoice
+
+config CACHE_ENABLE
+	bool "Enable cache"
+	default y
+	depends on SOC_SERIES_RISCV_TELINK_B91
+
+config HWDSP
```

Comment:
What exactly is HWDSP?  Is this unique to TELINK?  Maybe rename the symbol if so.

Reply:
HWDSP is specific for Telink B91 (NDS32) Toolchain, it enabled DSP commands in compiler.
Done, renamed to TELINK_B91_HWDP.